model/registry/db.yaml

```
@@ -268,16 +242,13 @@ groups:
           To be used instead of the generic `db.name` attribute.
         examples: [0, 1, 15]
         tag: tech-specific-redis
-      - id: sql.table
+      - id: collection.name
         type: string
         stability: experimental
-        brief: The name of the primary table that the operation is acting upon, including the database name (if applicable).
+        brief: The name of the collection (table, container) that the operation is applied to.
```

maybe, and move the part about operation to `trace/database.yaml` (`database-common.yaml` in the future)?
```suggestion
        brief: The name of a database collection (table, container).
```

---

```suggestion
        brief: The name of the collection within the database.
```